json-schemas/test/schema_test.ts

```diff
@@ -46,7 +46,7 @@ describe('Schema', () => {
     const validateAgainstSchema = (testCases: any[], schema: any, shouldFail = false) => {
         forEach(testCases, (testCase: any) => {
             const validationResult = validator.validate(testCase, schema);
-            const hasErrors = validationResult.errors.length !== 0;
+            const hasErrors = validationResult.errors && validationResult.errors.length !== 0;
```

Comment:
NIT: with TS upgraded we have nice things like [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) 🎉 
```suggestion
            const hasErrors = validationResult?.errors?.length !== 0;
```

Reply:
I get runtime errors with this though! Is it maybe my node version? I'm on 12 :/

Reply:
hmm, that's weird, it should _just work_ and compile to a `&&` chain 😕

Reply:
Oh, my bad. I added it when using typescript 3.0.1. Haven't tried it since bumping to 4.x.x